configs/siklu_80X0_defconfig

```diff
@@ -79,13 +85,3 @@ BR2_PACKAGE_TAR=y
 BR2_TARGET_ROOTFS_SQUASHFS=y
 BR2_TARGET_ROOTFS_SQUASHFS4_LZMA=y
 BR2_TARGET_ROOTFS_TAR_GZIP=y
-BR2_PACKAGE_LIBCURL=y
-BR2_PACKAGE_LIBCURL_CURL=y
-BR2_PACKAGE_LIBCURL_PROXY_SUPPORT=y
-BR2_PACKAGE_LIBCURL_COOKIES_SUPPORT=y
-BR2_PACKAGE_LIBCURL_EXTRA_PROTOCOLS_FEATURES=y
-BR2_PACKAGE_LIBCURL_OPENSSL=y
-BR2_PACKAGE_FMT=y
-BR2_PACKAGE_LIBFCGI=y
```

Comment:
@shmulik-aha to make sure we still support `BR2_PACKAGE_LIBFCGI`, can you please confirm that web UI works on with the SW that uses the new buildroot?

Reply:
@dima1308 I connected with web ui and saw that I can see local and remote devices. i will verify it again